.vscode/launch.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        // Dont forget to add the CONTENT variable to your environment.
+        {
+            "name": "Python: Upload AbuseDB",
+            "type": "python",
+            "request": "launch",
+            "module": "demisto_sdk",
+            "cwd": "${env:CONTENT}", // Content working directory
+            "args": [ // Run upload with AbuseDB
+                "upload",
+                "-i",
+                "${env:CONTENT}/Packs/AbuseDB/Integrations/AbuseDB"
+            ]
+        },
+        {
+            "name": "Python: Lint AbuseDB with logs",
+            "type": "python",
+            "request": "launch",
+            "module": "demisto_sdk",
+            "cwd": "${env:CONTENT}", // Content working directory
+            "args": [
+                "lint", // Lint AbuseDB
+                "-i",
+                "${env:CONTENT}/Packs/AbuseDB/Integrations/AbuseDB",
+                "-j",
+                "${env:CONTENT}/.demisto-sdk-report" // Write reports
+            ]
+        }
+    ]
+}
```

Comment:
@jochman - here is your problem.
![image](https://user-images.githubusercontent.com/50295826/127309943-3778e2b4-8eae-4103-a89c-0a36d936de20.png)